Source/Modules/lang.cxx

```diff
@@ -363,7 +363,7 @@ Language::Language():
 none_comparison(NewString("$arg != 0")),
 director_ctor_code(NewString("")),
 director_prot_ctor_code(0),
-symtabs(NewHash()),
```

Comment:
May as well also add in doxygenTranslator() to complete the list. Coding convention is one line per initializer please.

Reply:
@wsfulton Done!